py/torch_tensorrt/dynamo/_compiler.py

```diff
@@ -319,3 +325,122 @@ def compile_module(
         settings.use_fast_partitioner = True
 
     return partitioned_module
+
+
+def interpreter(
```

Comment:
Chose a better name for this function, this is not very descriptive. Function names should be a "verb" that conveys the action being performed by the function